leasing/models/rent.py

```diff
@@ -99,6 +99,65 @@ class Meta(NameModel.Meta):
         verbose_name_plural = pgettext_lazy("Model name", "Rent intended uses")
 
 
+class OldDwellingsInHousingCompaniesPriceIndex(TimeStampedModel):
```

Comment:
I just moved this model upward in the file so that it is defined on line 331 when the `old_dwellings_in_housing_companies_price_index` is being declared.